app/helpers/users/omniauth_callbacks_helper.rb

```diff
@@ -1,2 +1,5 @@
-module Users::OmniauthCallbacksHelper
+# frozen_string_literal: true
+
+# UsersOmniauthCallbacksHelper module
+module UsersOmniauthCallbacksHelper
```

Comment:
let's drop it, most helpers are evil ;) https://stackoverflow.com/questions/45468184/decorators-vs-helpers-in-rails